common/src/main/java/com/microsoft/identity/common/SharedPreferencesHelper.java

```diff
@@ -0,0 +1,90 @@
+package com.microsoft.identity.common;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+/**
+ * Convenience class for accessing {@link SharedPreferences}.
+ */
+public class SharedPreferencesHelper {
+
+    private final SharedPreferences mSharedPreferences;
+
+    /**
+     * Constructs an instance of SharedPreferencesHelper.
+     * The default operating mode is {@link Context#MODE_PRIVATE}
+     *
+     * @param context Interface to global information about an application environment.
+     * @param name    The desired {@link android.content.SharedPreferences} file. It will be created
+     *                if it does not exist.
+     */
+    public SharedPreferencesHelper(
+            final Context context,
+            final String name) {
+        mSharedPreferences = context.getSharedPreferences(name, Context.MODE_PRIVATE);
+    }
+
+    /**
+     * Constructs an instance of SharedPreferencesHelper.
+     *
+     * @param context       Interface to global information about an application enviroment.
+     * @param name          The desired {@link SharedPreferences} file. It will be created
+     *                      if it does not exist.
+     * @param operatingMode Operating mode {@link Context#getSharedPreferences(String, int)}.
+     */
+    public SharedPreferencesHelper(
+            final Context context,
+            final String name,
+            final int operatingMode) {
+        mSharedPreferences = context.getSharedPreferences(name, operatingMode);
+    }
+
+    /**
+     * Saves a Token (as a {@link String} to the {@link SharedPreferences} file.
+     *
+     * @param key   The name (key) of the Token to save.
+     * @param value The Token's value (as a {@link String}).
+     * @return True, if the value was successfully written.
+     */
+    public final boolean saveToken(
```

Comment:
I would suggest making this really generic... and having putString... we may end up using this class in the future for something other than Tokens

Reply:
Sounds good - easy change 👍